### ISSUE TITLE
`DataSampleDir` in `FITS.unitTest` doesn't resolve to something existing

### ISSUE THREAD

**Issue by EastEriq** [EranOfek/AstroPack#396]
Just saying. I was caressing the idea of adding lines to test the sanity of my latest additions (#394 and #395) in `FITS.unitTest`, but on the installation I'm on 
```
>> FITS.unitTest
12:12:50.070 [TST] FITS.unitTest sarted
Error using cd
Unable to change current folder to
'/raid/enrico/AstroPack/matlab/util/+tools/+os/../../../../../data/TestImages/unitTest'
(Name is nonexistent or not a folder).

Error in FITS.unitTest (line 11)
	cd(DataSampleDir);
 ```
giving up, will test separately.

**Comment by agioffe** [comment]
If you use the Installer class and choose to install TestImagesAstroPack, this directory should be created with some stuff inside.

**Comment by EastEriq** [comment]
Thanks. so:
```
>> I=Installer

I = 

  Installer with properties:

       Items: [1×1 struct]
    UserData: []

>> I.install('TestImagesAstroPack')
16:44:23.419 [INF] install started: items to install: 1
16:44:23.606 [INF] creating folder: ~/matlab/data/TestImages/unitTest
16:44:39.291 [ERR] installSingle failed: TestImagesAstroPack
16:44:39.397 [INF] install: done
>> FITS.unitTest
16:45:19.380 [TST] FITS.unitTest sarted
Error using cd
Unable to change current folder to
'/raid/enrico/AstroPack/matlab/util/+tools/+os/../../../../../data/TestImages/unitTest'
(Name is nonexistent or not a folder).

Error in FITS.unitTest (line 11)
	cd(DataSampleDir);
```
in my $HOME, `~/matlab/data/TestImages/unitTest/` has been created but is empty, and `DataSampleDir` is still wrong. 
What do I know...

**Comment by EastEriq** [comment]
I see that `I.seeAvailableData` prints out that `'TestImagesAstroPack'` should come from `http://euler1.weizmann.ac.il/archive/AstroPack/data/TestImages/unitTest/index.html`, but euler1 is offline.

**Comment by agioffe** [comment]
Please, do git pull. The address has been changed in Installer.yml.

**Comment by EastEriq** [comment]
Ok, that corrects the link and does install. However, `FITS.unitTest` still fails because of `getTestDataDir` which makes a very rigid assumption about where this data should sit.

**Comment by agioffe** [comment]
I have fixed the **getTestDataDir**, but **FITS.unitTest** still fails for me, because it cannot read the second (!) file in the list set by "*.fits". There is something with the class file itself, that I do not understand.

**Comment by EastEriq** [comment]
Not ok. `Installer.install` puts data in `~/matlab/data/TestImages/unitTest`, whereas `DataSampleDir` points  to [AstroPackDir]+`../data/TestImages/unitTest`. That is not the same in my installation (and I'm not aware that it is mandatory to install AstroPack in `~/matlab`).

**Comment by agioffe** [comment]
OK, to make it consistent this way I changed the **FITS.unitTest** so that this path is tuned to what the **Installer** class knows about it. **getTestDataDir** is no more used.

**Comment by EastEriq** [comment]
> I have fixed the **getTestDataDir**, but **FITS.unitTest** still fails for me, because it cannot read the second (!) file in the list set by "*.fits". There is something with the class file itself, that I do not understand.

With the DataDir correction I'm now able to see the problem, it has to do with a recursive call of the FITS class constructor, when a list of filenames are given. I'm solving it, but since it is touching something essential, I'll do that on the branch `FITSheaderWithLongStrings` which I just opened a few days ago. After all, this issue is born exactly because I got into adding the unit test for my changes.
